paper.bib

```diff
@@ -19063,6 +19063,151 @@ @Article{Thrampoulidis2015LASSO
+  Volume                   = {45}
+}
+
+@Book{Mackay2003,
+  Title                    = {Information theory, inference and learning algorithms},
+  Author                   = {MacKay, D.},
+  Publisher                = {Cambridge university press},
+  Year                     = {2003}
+}
+
+@Article{Shimodaira2000,
+  Title                    = {Improving predictive inference under covariate shift by weighting the log-likelihood function},
+  Author                   = {Shimodaira, A.},
+  Journal                  = {Journal of Statistical Planning and Inference},
+  Year                     = {2000},
+  Number                   = {2},
+  Pages                    = {227 - 244},
+  Volume                   = {90},
+
+  Doi                      = {https://doi.org/10.1016/S0378-3758(00)00115-4},
+  ISSN                     = {0378-3758},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0378375800001154}
+}
+
+@Article{Sugiyama2008,
+  Title                    = {Direct Importance Estimation for Covariate Shift Adaptation},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Nakajima, S. and Kashima, H. and von B\"unau, P. and Kawanabe, M.},
+  Journal                  = {Annals of the Institute of Statistical Mathematics},
+  Year                     = {2008},
+  Number                   = {4},
+  Pages                    = {699--746},
+  Volume                   = {60}
+}
+
+@Article{SUGIYAMA2011735,
+  Title                    = {Least-squares two-sample test},
+  Author                   = {M. Sugiyama and T. Suzuki and Y. Itoh and T. Kanamori and M. Kimura},
+  Journal                  = {Neural Networks},
+  Year                     = {2011},
+  Number                   = {7},
+  Pages                    = {735 - 751},
+  Volume                   = {24},
+
+  Doi                      = {https://doi.org/10.1016/j.neunet.2011.04.003},
+  ISSN                     = {0893-6080},
+  Keywords                 = {Two-sample test, Homogeneity test, Density ratio estimation, Unconstrained least-squares importance fitting, Pearson divergence},
+  Url                      = {http://www.sciencedirect.com/science/article/pii/S0893608011001262}
+}
+
+@Article{Sugiyama2011,
+  Title                    = {Least-squares two-sample test},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Itoh, Y. and Kanamori T. and Kimura M.},
+  Journal                  = {Neural Networks},
+  Year                     = {2011},
+  Number                   = {7},
+  Pages                    = {735 - 751},
+  Volume                   = {24}
+}
+
+@Book{Sugiyama2012,
+  Title                    = {Density Ratio Estimation in Machine Learning},
+  Author                   = {Sugiyama, M. and Suzuki, T. and Kanamori, T.},
+  Publisher                = {Cambridge University Press},
+  Year                     = {2012},
+
+  Location                 = {Cambridge, UK}
+}
+
+@Article{sugiyama2007covariate,
```

Comment:
@Article{Sugiyama2007Covariate,
  author     = {Sugiyama, Masashi and Krauledat, Matthias and M\"{u}ller, Klaus-Robert},
  title      = {Covariate Shift Adaptation by Importance Weighted Cross Validation},
  journal    = {J. Mach. Learn. Res.},
  year       = {2007},
  volume     = {8},
  pages      = {985--1005},
  month      = dec,
  issn       = {1532-4435},
  acmid      = {1390324},
  issue_date = {12/1/2007},
  numpages   = {21},
  publisher  = {JMLR.org},
  url        = {http://dl.acm.org/citation.cfm?id=1314498.1390324},
}